pkg/apis/v1/nodepool_conversion_test.go

```diff
@@ -269,6 +270,24 @@ var _ = Describe("Convert V1 to V1beta1 NodePool API", func() {
 			Expect(v1beta1nodepool.Status.Resources[resource]).To(Equal(v1nodepool.Status.Resources[resource]))
 		}
 	})
+	Context("Round Trip", func() {
+		DescribeTable(
+			"NillableDuration",
+			func(value string, field func() *NillableDuration) {
+				str := fmt.Sprintf("%q", value)
```

Comment:
curious why this needs to be quotes? does it do anything differently?
```suggestion
				str := fmt.Sprintf("%s", value)
```

Reply:
If you don't quote it, it will fail to parse, I believe because it's trying to parse it as a number rather than a string literal.